Python/To-do List/README.md

```diff
@@ -1 +1,42 @@
-
+﻿
+
+README.md
+
+Description:
+
+With the rapid development of the world, everyone is engaged in a lot of work or even stuck with
+
+the thought of where to start from. To resolve this customary issue here presents ‘ON IT!!’.It is a
+
+python based GUI that helps boost productivity and helps get our work done in an organized
+
+way. It includes 4 fields to note the tasks to be performed today, tomorrow, the current week, or
+
+users can customize the date with their own will.
+
+Tech-Stack:
+
+Python, Tkinter
+
+Video:
+
+[Video](https://drive.google.com/file/d/19s8E6qCUjR9Du1qNaTrbOPS3vwytF1es/view?usp=sharing)[ ](https://drive.google.com/file/d/19s8E6qCUjR9Du1qNaTrbOPS3vwytF1es/view?usp=sharing)[Drive](https://drive.google.com/file/d/19s8E6qCUjR9Du1qNaTrbOPS3vwytF1es/view?usp=sharing)[ ](https://drive.google.com/file/d/19s8E6qCUjR9Du1qNaTrbOPS3vwytF1es/view?usp=sharing)[Link](https://drive.google.com/file/d/19s8E6qCUjR9Du1qNaTrbOPS3vwytF1es/view?usp=sharing)
+
+Basic Components:
+
+Python-Loops,Functions,etc
+
+Tkinter-PhotoImage, Label-Frames, Entry box, Button, icon bitmap
+
+Steps to Run in Local:
+
+\1. Download the folder name ‘To-do list’
+
+\2. Open the folder in any IDE which supports python (used Version- Python 3.10,
+
+IDE-Pycharm)
+
+\3. Install pillow library by running the command- pip install pillow in the terminal
+
+\4. Run the program

```

Comment:
Remove \ before the numbers

Reply:
Also, i would suggest to go through this article [Readme](https://medium.com/analytics-vidhya/how-to-create-a-readme-md-file-8fb2e8ce24e3) like how to write readme file with proper Syntax.